src/ACPAnalyticsAndroidUnitTests/ACPAnalyticsAndroidUnitTests.cs

```diff
@@ -42,7 +42,7 @@ public void Setup()
         public void GetACPAnalyticsExtensionVersion_Returns_CorrectVersion()
         {
             // verify
-            Assert.That(ACPAnalytics.ExtensionVersion(), Is.EqualTo("1.2.6"));
+            Assert.That(ACPAnalytics.ExtensionVersion(), Is.EqualTo("1.2.9"));
```

Comment:
should the latest 1.2.10 analytics release be used?

Reply:
oh yeah, it was released last night.  It fixed an assurance integration issue which no debug flag showing up in some events.

Reply:
Will updated, created this PR yesterday so might have missed it